metadata-ingestion/tests/unit/api/source_helpers/test_auto_browse_path_v2.py

```diff
@@ -0,0 +1,448 @@
+from itertools import zip_longest
+from typing import Any, Dict, Iterable, List
+from unittest.mock import patch
+
+import datahub.metadata.schema_classes as models
+from datahub.emitter.mce_builder import (
+    make_container_urn,
+    make_dataplatform_instance_urn,
+    make_dataset_urn,
+)
+from datahub.emitter.mcp import MetadataChangeProposalWrapper
+from datahub.emitter.mcp_builder import DatabaseKey, SchemaKey
+from datahub.ingestion.api.source_helpers import (
+    _prepend_platform_instance,
+    auto_browse_path_v2,
+    auto_status_aspect,
+)
+from datahub.ingestion.api.workunit import MetadataWorkUnit
+from datahub.ingestion.source.sql.sql_utils import gen_schema_container
+from datahub.metadata.schema_classes import BrowsePathEntryClass, BrowsePathsV2Class
+
+
+def test_auto_browse_path_v2_gen_containers_threaded():
```

Comment:
This and `_iterate_wus_round_robin` helper are the only new tests. Everything else copied over from `test_source_helpers.py`

Reply:
Yes, this is fine.